spec/services/fraud_review_check_spec.rb

```diff
@@ -91,5 +91,25 @@
 
       it { expect(subject.fraud_review_eligible?).to eq(false) }
     end
+
+    context 'fraud review is pending for passed ipp enrollment' do
+      let(:user) { create(:user) }
+      let!(:profile) { create(:profile, fraud_review_pending_at: 1.day.ago, user: user) }
+      let!(:enrollment) { create(:in_person_enrollment, :passed, user: user, profile: profile) }
+
+      it { expect(subject.ipp_fraud_review_pending?).to eq(true) }
```

Comment:
not blocking but can we update this to look more like lines 110-112 for increased readability